packages/override-tools/src/Cli.ts

```diff
@@ -268,18 +400,30 @@ async function readManifest(file: string, version?: string): Promise<Manifest> {
     throw ex;
   }
 
-  const gitReactRepo = await GitReactFileRepository.createAndInit();
-
-  const ovrPath = path.dirname(file);
-  const ovrFilter = /^.*\.(js|ts|jsx|tsx|cpp|h)$/;
-  const ovrRepo = new OverrideFileRepositoryImpl(ovrPath, ovrFilter);
-
   const rnVersion = version || (await getInstalledRNVersion(file));
-  const reactRepo = FileRepository.bindVersion(gitReactRepo, rnVersion);
+  const reactRepo = bindVersion(versionedReactRepo, rnVersion);
 
   return new Manifest(data, ovrRepo, reactRepo);
 }
 
+/**
+ * Wraps the function in a try/catch, failing the spinner if an exception is
+ * thrown to allow unmangled output
+ */
+async function spinnerGuard<T>(
+  spinner: ora.Ora,
+  fn: () => Promise<T>,
+): Promise<T> {
+  try {
+    return await fn();
+  } catch (ex) {
+    if (spinner.isSpinning) {
+      spinner.fail();
+    }
+    throw ex;
```

Comment:
I think this might not get propagated correctly. Need to test this out.

Reply:
This is propagated correctly. Node just displays its own exceptions in a weird way, and our console.error's were interfering.